packages/backend/src/graphql/schema.graphql

```diff
@@ -1,9 +1,34 @@
 type Query {
   account(id: String!): Account
+
+  "Fetch a page of accounts."
+  accounts(input: PaginationInput): AccountsConnection!
```

Comment:
Careful using input types for queries. This breaks the Relay pagination specification and might mean anyone using the relay client wouldn't get the built in caching functionality for pagination. Anyone using apollo might also need to specify caching functions manually for apollo instead of using the [built in utility](https://www.apollographql.com/docs/react/pagination/cursor-based/#relay-style-cursor-pagination).